src/shared/user-record-show.c

```diff
@@ -206,6 +212,31 @@ void user_record_show(UserRecord *hr, bool show_full_group_info) {
         if (hd)
                 printf("   Directory: %s\n", hd);
 
+        if (hr->blob_directory) {
+                const char *filename;
+                const uint8_t *hash_bytes;
+                unsigned i = 0, n;
+
+                printf("   Blob Dir.: %s\n", hr->blob_directory);
+
+                n = hashmap_size(hr->blob_manifest);
+                HASHMAP_FOREACH_KEY(hash_bytes, filename, hr->blob_manifest) {
+                        _cleanup_free_ char *path = NULL, *link = NULL, *hash = NULL;
+
+                        path = path_join(hr->blob_directory, filename);
+                        if (path)
```

Comment:
frankly, i'd just do:

```c
if (!path)
        return (void) log_oom();
```

Reply:
I'm for it, but the rest of the function isn't written that way and it just silently continues through any error it encounters. Are you sure you want this to be an exception?

Reply:
hmm. i guess it doesn't matter, just leave it as is